_data/vendors.yml

```diff
@@ -75,6 +75,14 @@
   pricing_source: https://coderpad.io/pricing
   updated_at: 2019-06-28
 
+- name: Docker
+  url: https://www.docker.com/
+  base_pricing: $7 per u/m
+  sso_pricing: $21 per u/m
+  percent_increase: 33%
```

Comment:
This is a 300% increase (if one isn't pedantic with per _cent_ but that's another review)